mix.exs

```diff
@@ -42,11 +42,11 @@ defmodule Scrivener.Ecto.Mixfile do
   defp deps do
     [
       {:scrivener, "~> 2.0"},
-      {:ecto, "~> 2.0"},
+      {:ecto, "~> 2.1"},
```

Comment:
We  can't make this more restrictive without bumping a major version, this should stay at `~> 2.0`.